options.ts

```diff
@@ -154,36 +154,49 @@ export class OptionsBase {
 	// ex, "$ <cli name> emulate android --profile-dir" will add profile-dir to yargs.argv as profile-dir and profileDir
 	// IMPORTANT: In your code, it is better to use the value without dashes (profileDir in the example).
 	// This way your code will work in case "$ <cli name> emulate android --profile-dir" or "$ <cli name> emulate android --profileDir" is used by user.
-	private getSecondaryOptionName(optionName: string): string {
+	private getNonDashedOptionName(optionName: string): string {
 		let matchUpperCaseLetters = optionName.match(/(.+?)([-])([a-zA-Z])(.*)/);
 		if (matchUpperCaseLetters) {
 			// get here if option with upperCase letter is specified, for example profileDir
 			// check if in knownOptions we have its kebabCase presentation
 			let secondaryOptionName = util.format("%s%s%s", matchUpperCaseLetters[1], matchUpperCaseLetters[3].toUpperCase(), matchUpperCaseLetters[4] || '');
-			return this.getSecondaryOptionName(secondaryOptionName);
+			return this.getNonDashedOptionName(secondaryOptionName);
+		}
+
+		return optionName;
+	}
+
+	private getDashedOptionName(optionName: string): string {
+		let matchUpperCaseLetters = optionName.match(/(.+?)([A-Z])(.*)/);
```

Comment:
You can assign the regex to a `static` variable so that it's not created each time we enter this method. This is valid for `getNonDashedOptionName` too